src/main/java/org/isf/config/SecurityConfig.java

```diff
@@ -134,6 +134,36 @@ protected void configure(final HttpSecurity http) throws Exception {
 				.antMatchers(HttpMethod.DELETE, "/wards/**").hasAuthority("admin")
 				.antMatchers(HttpMethod.PATCH, "/wards/**").hasAuthority("admin")
 				.antMatchers(HttpMethod.GET, "/wards/**").hasAnyAuthority("admin", "guest")
+				.antMatchers(HttpMethod.POST, "/exams/**").hasAuthority("admin")
```

Comment:
We need it for all controllers.

We could use a regular expression to reduce the number of `antMatchers` required.

See: https://docs.spring.io/spring-security/site/docs/4.1.1.RELEASE/apidocs/org/springframework/security/config/annotation/web/AbstractRequestMatcherRegistry.html#regexMatchers-org.springframework.http.HttpMethod-java.lang.String...-

Reply:
Yes, this method is very verbose for all controllers. 
IMHO regex probably introduce less code maintainability, what do you think?

Reply:
I think we can tackle this problem in a second step, @emecas and @vicusotto do you agree?

Reply:
yes , we can leave this for a different stage during API implementation, if new code that you are adding here to SecurityConfig.java  is not affecting or representing changes for the current and future test suite(s) implementation ( I don't think so), I would give an approve to this PR.

Reply:
> is not affecting or representing changes

I think so, anyway I'm going to test it as well

Reply:
you are rigth, i just confirmed on my side too!!!